game.js

```diff
@@ -28,17 +39,53 @@ const quitGame = () => {
  * @returns {undefined}
  */
 const gameLoop = () => {
+console.log("I have a random number in mind")
+console.log("It's between 1 and 1000")
+console.log("You have 10 guesses total")
+let randomNum = generateRandomNumber()
+let guess;
+
+
```

Comment:
while loop would be better. While count > 0. Then you can decrement count depending on the result